target/linux/mediatek/dts/mt7981b-openwrt-one-mikrobus-spidev.dtso

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/*
+ * Copyright (C) 2025
+ * Author: Mark Birss <mark.birss@gmail.com>
+ */
+
+/dts-v1/;
+/plugin/;
+
+/ {
+        compatible = "openwrt,one", "mediatek,mt7981";
+        status = "okay";
+
+        fragment@0 {
+                target-path = "/soc/spi@1100b000";
+                __overlay__ {
+                spidev@0 {
+                        compatible = "linux,spidev";
```

Comment:
maybe silabs,si3210 to get autoload? There are 3 devices in tree which use this hack: D-Link DWR-512 B, GL.iNet GL-B2200, GL.iNet GL-S1300